astropy/__init__.py

```diff
@@ -234,3 +234,6 @@ def online_help(query):
         del locals()[varname]
 
 del varname, __module_type__
+
+
+from astropy.utils.system_info import system_info
```

Comment:
Does this also need to be updated?

https://github.com/astropy/astropy/blob/e7a556fdeda076f72277b06a8ef6bf9acc29b8f4/astropy/__init__.py#L199

Reply:
you tell me. I have never seen this variable name before, I don't know what it does.

Reply:
Looks like @saimn renamed it from `__dir__` in https://github.com/astropy/astropy/pull/7617 . Do you remember why we need this, Simon?

Reply:
Yes, `module.__dir__` has a special meaning since Python 3.7: https://docs.python.org/3.12/reference/datamodel.html#customizing-module-attribute-access

Reply:
Oh, I see. I think my new function *does* belong in `__dir_inc__`, which lists everything that should *not* be dropped from the namespace, but since I added it at the end of the module, it's only included after namespace cleanup happens, so it dodges that. I think the cleanup ought to be the last thing to happen in the module, so I'll move my import and update this list, thanks !